docs/topics/running/statistics/8877-metrics.md

```diff
@@ -57,11 +57,10 @@ with Prometheus and Grafana](../../../../howtos/prometheus).
 
 #### Sample dashboard
 
-We provide a [sample Grafana dashboard][`8877-metrics-grafana.json`]
+We provide a [sample Grafana dashboard](https://grafana.com/dashboards/10434)
```

Comment:
Wait, where does this `10434` come from?

Reply:
Good catch, thanks. Started to forget which ID I was using with 3 of them in my head now. New standard should be: default to using ID: 13758, keep one link to ID: 4698 at "additional install options" and removing all references to ID: 10434.